client/src/Ratings/NewReviewForm.jsx

```diff
@@ -1,189 +1,140 @@
-import React from "react";
+import React, {useState, useRef} from "react";
 import Body from "./NewReviewPages/Body.jsx";
 import Characteristics from "./NewReviewPages/Characteristics.jsx";
 import ReviewPhotos from "./NewReviewPages/ReviewPhotos.jsx";
 import Summary from "./NewReviewPages/Summary.jsx";
 import UserInfo from "./NewReviewPages/UserInfo.jsx";
 const axios = require("axios");
 
-class NewReviewForm extends React.Component{
-  constructor(props) {
-    super(props);
-    this.state = {
-      page: 1,
-      rating: 0,
-      recommended: null,
-      characteristics: {},
-      summary: '',
-      body: '',
-      photos: [],
-      reviewer_name: '',
-      reviewer_email: ''
-    }
-    this.handleClick = this.handleClick.bind(this);
-    this.setStarRating = this.setStarRating.bind(this);
-    this.setRecommended = this.setRecommended.bind(this);
-    this.setChars = this.setChars.bind(this);
-    this.setSummary = this.setSummary.bind(this);
-    this.setBody = this.setBody.bind(this);
-    this.setPhotos = this.setPhotos.bind(this);
-    this.setUserInfo = this.setUserInfo.bind(this);
-    this.sendPostRequest = this.sendPostRequest.bind(this);
-  }
+function NewReviewForm(props) {
+  const [page, setPage] = useState(1);
+  const [rating, setRating] = useState(0);
+  const [recommended, setRecommended] = useState(null);
+  const [characteristics, setCharacteristics] = useState(null);
+  const [summary, setSummary] = useState('');
+  const [body, setBody] = useState('');
+  const [photos, setPhotos] = useState(null);
+  const [name, setName] = useState('');
+  const [email, setEmail] = useState('');
 
-  componentDidUpdate() {
 
-  }
-  setChars(value){
-    this.setState({
-      characteristics: value
-    }, ()=>{
-      console.log(this.state.characteristics)
-    })
+  let ratingRef = useRef(rating);
+  let recommendedRef = useRef(recommended);
+  let characteristicsRef = useRef(characteristics);
+  let bodyRef = useRef(body);
+  let nameRef = useRef(name);
+  let emailRef = useRef(email);
 
-  }
-  handleClick(value){
-    if(this.state.page > 0 || this.state.page < 8) {
-      this.setState({
-        page: this.state.page + value
-      })
-    }
-  }
+  // sendPostRequest(obj) {
+  //   let params = {product_id: 40344};
+  //   let data = this.state;
+  //   axios.post('/reviews', {params: params, data: data})
```

Comment:
update params to be dynamic. also data is still referencing this.state. Is this being used?